Bottom: 7d11f01d4de3d43aa7c303dceaee0cfbb6411add
Top:    525babe7d9fcc6004261f4e581a376f0f37f0d3b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-24 22:32:17 +0100

Refresh of hid-gtk-refactor-viewport-hand

---

diff --git a/src/hid/gtk/gtkhid-gdk.c b/src/hid/gtk/gtkhid-gdk.c
index 52c5860..aefda85 100644
--- a/src/hid/gtk/gtkhid-gdk.c
+++ b/src/hid/gtk/gtkhid-gdk.c
@@ -669,7 +669,7 @@ ghid_fill_circle (hidGC gc, int cx, int cy, int radius)
   gint w, h, vr;
   render_priv *priv = gport->render_priv;
 
-  w = gport->width *  priv->view.pcb_units_per_pixel;
+  w = gport->width  * priv->view.pcb_units_per_pixel;
   h = gport->height * priv->view.pcb_units_per_pixel;
   if (SIDE_X (cx) < gport->view_x0 - radius
       || SIDE_X (cx) > gport->view_x0 + w + radius
diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 10db8db..8ae17f0 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1073,7 +1073,7 @@ ghid_pinout_preview_expose (GtkWidget *widget,
 
   gport->width = da_w;
   gport->height = da_h;
-  gport->view_width = da_w * priv->view.pcb_units_per_pixel;
+  gport->view_width = da_w  * priv->view.pcb_units_per_pixel;
   gport->view_height = da_h * priv->view.pcb_units_per_pixel;
   gport->view_x0 = (pinout->x_max - gport->view_width) / 2;
   gport->view_y0 = (pinout->y_max - gport->view_height) / 2;
